doc/manual/index.rst

```diff
@@ -1,136 +0,0 @@
-:orphan:
-
-.. _manual:
-
-User Manual
-===========
```

Comment:
did you remove this page on purpose?

Reply:
Yes. It's not linked to from anywhere, as `doc/documentation.rst` effectively serves as our manual now

Reply:
the only things that weren't in `documentation.rst` I added:

https://github.com/mne-tools/mne-python/pull/4743/files#diff-9419719ec9ef1acef6f3fbada68f1d89